build.gradle

```diff
@@ -175,8 +175,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
Cool, could you please help me here? As I said, I tried adding the dependency within the module but I didn't work